src/syntax/tokenizer.rs

```diff
@@ -329,7 +331,9 @@ impl<'s, T: Borrow<Tokenizer>> TokenizerI<'s, T> {
                 }
                 _ => {
                     let token = self.scan_word()?;
-                    if token.kind == TokenKind::When || token.kind == TokenKind::Given {
+                    if token.kind == TokenKind::When || token.kind == TokenKind::Given
+                    // || token.kind == TokenKind::It // @follow-up - I think this is needed, but it breaks the tests because of invalid {} character in emit event statement
```

Comment:
This is not correct. Actions can contain any character because when emitted they are just comments.

Could you expand on why it's needed?

Reply:
Ah, yes. Ignore this, apologies - I originally thought identifier mode was required for scanning all `Word` tokens but then equally saw that other characters in the actions would then be considered invalid, so this makes sense given that they are emitted as comments.

Reply:
No need to apologize at all!